test/game_statistics_test.rb

```diff
@@ -5,9 +5,9 @@
 class GameStatisticsTest < MiniTest::Test
 
   def setup
-    game_path = './game_stats_fixtures/games_fixtures.csv'
-    team_path = './game_stats_fixtures/teams_fixtures.csv'
-    game_teams_path = './game_stats_fixtures/game_teams_fixtures.csv'
+    game_path = './data/games.csv'
```

Comment:
We are moving towards a module and these won't exist in the next refactoring phase.  It will be moved into Stat_Tracker